tools/runtime-stats-monitor.md

```diff
@@ -0,0 +1,215 @@
+
+:::note
+The default setting for the Display Elements, Panel Settings, and Styling are greyed out and not adjustable by default. To make modifications or build your own, you will need to create your own configuration presets. Instructions for each customization are below.
+:::
+
+### Display Elements
+
+Each Display Element displays one or more stats that can be added, removed, or modified under the element's **Stats** field.
+
+To use the default Display Elements:
+
+1. It may already be the default in the **Configuration** field of the **Inspector** tab of the RNSM component.
+
+OR
+
+1. From the **Project** tab, go to **Packages** > **Multiplayer Tools** > **NetStatsMonitor** > **Assets** > **Configurations**. The default configuration for the RNSM is in this folder.
+2. Select the `GameObject` with the RNSM component so that it appears in the **Inspector** tab.
+3. Drag the default configuration asset from the **Project** tab to the **Configuration** field under the **RuntimeNetStatsMonitor** component of your `GameObject`.
+
+To configure your own Display Elements presets:
+
+1. Select the **Assets** folder under the **Project** tab.
+2. Right-click in the **Assets** folder, then navigate to **Create** > **Multiplayer** and click on **NetStatsMonitorConfiguration**.
+3. The configuration script appears under the **Assets** folder and also opens in the **Inspector** tab.
+4. From the **Inspector** tab, you can use the + or - symbols to add each Display Element you want to include in this preset. Each field is configurable from here.
+5. Once configured, you can add your Display Elements configuration preset to your RNSM `GameObject` by selecting the the RNSM `GameObject` to appear in the **Inspector** tab.
+6. Your **Assets** folder should still be visible in the **Project** tab, so select your newly created configuration preset to drag and drop over the **Configuration** field under the **RuntimeNetStatsMonitor** component in the **Inspector** tab.
+7. **Save** your scene.
+
+### Panel Settings
+
+While working in the Unity Editor with the Runtime Network Stats Monitor, you may find your screen real estate to be difficult to work with. To help remedy these situations, you can use the default panel settings or create your own to set up your workspace exactly as you want it.
+
+To use the default Panel Settings:
+
+1. From the **Project** tab, go to **Packages** > **Multiplayer Tools** > **NetStatsMonitor** > **Implementation** > **UI** > **Resources**. The default panel settings configuration for the RNSM is in this folder: `UnityMpToolsRnsmDefaultPanelSettings.asset`.
+2. Select the `GameObject` with the RNSM component so that it appears in the **Inspector** tab.
+3. Drag the default panel setting asset from the **Project** tab to the **Panel Settings Override** field under the **RuntimeNetStatsMonitor** component of your `GameObject`.
```

Comment:
Leaving this override empty should be sufficient to use the default. It shouldn't be necessary to drag the default into the override 🙂